components/effortless-chef-io/welcome.md

```diff
@@ -0,0 +1,16 @@
+# Welcome to the Chef Effortless Patterns
+
+The Effortless Pattern is a way to better manage Chef Infra and Chef InSpec workloads using Chef Habitat and visualizing your fleet using Chef Automate. We believe that each of these technologies working together can help you better manage your infrastructure.
+
```

Comment:
1) Who should use the Effortless pattern? Along the lines of "If you use Chef Infra and Chef Inspec workloads to do X...
2) What are the advantages of using Effortless?? Be as specific as possible

Reply:
I added two more sections under this one that I think might answer this. Let me know what you think.